sct-commons/src/main/java/org/lfenergy/compas/sct/commons/service/impl/ExtRefService.java

```diff
@@ -2,43 +2,37 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-package org.lfenergy.compas.sct.commons.scl;
+package org.lfenergy.compas.sct.commons.service.impl;
```

Comment:
don't use an "`impl`" package inside `service` but rather use an "`api`" package for interfaces inside `commons` package

Reply:
ok